src/viewer_process.py

```
@@ -1,41 +1,46 @@
     grid = Grid(memoria)
+    robo = Robo(
+        robo_id,
+        grid,
+        memoria.robots_info,
+        {
+            'grid_mutex': memoria.grid_mutex,
+            'robots_mutex': memoria.robots_mutex,
+            'battery_mutexes': memoria.battery_mutexes
+        }
+    )
+    robo.run()
```

Pylint (3.3.7)

[error] Instance of 'Robo' has no 'run' member

(E1101)